src/renderer/components/cluster-manager/cluster-view.tsx

```diff
@@ -65,6 +67,7 @@ export class ClusterView extends React.Component {
         initView(clusterId); // init cluster-view (iframe), requires parent container #lens-views to be in DOM
         requestMain(clusterActivateHandler, clusterId, false); // activate and fetch cluster's state from main
         catalogEntityRegistry.activeEntity = catalogEntityRegistry.getById(clusterId);
+        this.store.setActive(clusterId);
```

Comment:
Setting active cluster was missing somehow at all (only in `ClusterStore.fromStore`)

Reply:
If we keep only this line and remove all other changes from the PR then cluster commands are shown, but navigation to corresponding pages is broken for some reasons..

Reply:
Why do we need this, shouldn't it be possible to derive the active cluster from the active catalog entity?

Reply:
I'm also wondering do we really need this?

Reply:
Question is why do we need `ClusterStore.setActive(id: string)`? 
It sets `ClusterStore.activeCluster` and used in:

```ts
  @computed get active(): Cluster | null {
    return this.getById(this.activeCluster);
  }
```

Reply:
Should we use something like this instead?

```ts
  @computed get active(): Cluster | null {
    return this.getById(catalogEntityRegistry.activeEntity?.getId());
  }
```

Reply:
I would get rid of `active()` from `ClusterStore` entirely. Since it has no use on `main` and from `renderer` it would be better to just use `ClusterStore.getInstance().getById(catalogEntityRegistry.activeEntity?.getId())` directly.